test/classes.coffee

```diff
@@ -1574,3 +1670,76 @@ test "CS6 Class extends a CS1 compiled class with super()", ->
   eq B.className(), 'ExtendedCS1'
   b = new B('three')
   eq b.make(), "making a cafe ole with caramel and three shots of espresso"
+
+test 'Bound method called as callback before binding throws runtime error', ->
```

Comment:
Checking for errors goes in `error_messages.coffee`. Not sure why, we should probably refactor them out of there into the relevant test files, but for now that’s where all such tests go.